src/v1/base/TopNavigation.tsx

```diff
@@ -8,15 +8,15 @@ type ItemProps = TopNavigationProps;
 
 const TopNavigation = ({ children }: TopNavigationProps) => {
   return (
-    <div className="relative flex h-[2.4rem] w-full items-center justify-center bg-opacity-0 px-[2rem] py-[1.7rem] text-md">
+    <div className="fixed left-0 top-[0rem] z-50 flex h-[2.4rem] w-[calc(100%-0rem)] items-center justify-center bg-white px-[4rem] py-[3.7rem] text-md">
       {children}
     </div>
```

Comment:
p1;

<img width="720" alt="image" src="https://github.com/prgrms-web-devcourse/Team-Gaerval-Dadok-FE/assets/73218463/c6aa1099-26cc-4b49-9a20-ba67a4e38808">

위 사진과 같이 TopNavigation이 양옆으로 튀어나오는 현상이 발생하고있어요 😢 

```suggestion
    <div className="fixed left-0 right-0 top-[0rem] z-50 mx-auto flex h-[2.4rem] w-[calc(100%-0rem)] max-w-[43rem] items-center justify-center bg-white px-[4rem] py-[3.7rem] text-md">
      {children}
    </div>
```

다음과 같이 `mx-auto`, `right-0`, `max-w-[43rem]`을 추가한다면 어느 환경에서든 잘 표현될 수 있을 것 같습니다!